e2e_test/batch/join/issue_7115.slt

```diff
@@ -0,0 +1,28 @@
+# https://github.com/risingwavelabs/risingwave/issues/7115
+
+statement ok
+create table t (id int);
+
+statement ok
+create materialized view v as select count(*) cnt from t;
+
+statement ok
+SET QUERY_MODE TO local;
```

Comment:
Just rename the suffix of this file to `.slt.part`, and it will run in both local mode and distributed mode.

Reply:
Haven't fixed for `distributed_mode` yet unfortunately. That's why use this as workaround for now.

Reply:
Oh just got what you mean after thinking about it. Shall change it.

Reply:
Please add some data to verify it.